docs/source/evaluation_scripts.md

```diff
@@ -129,3 +129,205 @@ Here are the steps to configure remote evaluation:
+                    )
+                )
+        ```
+
+        You can modify the relevant parts of the environment service in order to make it work for your case. For example, `feedback` can totally depend on the results obtained from the `env.env` after performing an action, which in-turn depends on the sensors expected from the agent. In our example, `feedback[2]` is the stop signal sensor. The `unpack_for_grpc` and `pack_for_grpc` are methods to receive and send messages with the agent.
+
+        This is a high-level description of the class and the implementations may vary on a case-by-case basis.
+    3. _Edit the requirements file_: Change the [requirements file](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/requirements/environment.txt) according to the packages required by your environment.
+
+    4. _Edit environment Dockerfile_: You may choose to modify the [Dockerfile](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/docker/environment/Dockerfile) which will host the Environment service on a GRPC server.
+
+    5. _Edit the docker environment variables_: Fill in the following information in the [`docker.env`](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/docker/environment/docker.env) file:
+
+        ```env
+        AUTH_TOKEN=<Add your EvalAI Auth Token here>
+        EVALAI_API_SERVER=<https://eval.ai>
+        LOCAL_EVALUATION = True
+        QUEUE_NAME=<Go to the challenge manage tab to get challenge queue name.>
+        ```
+
+    6. _Create the docker image and host it_: Create an environment docker image using the above steps using `docker build` command and host it.
+
+    7. _Add environment image the challenge configuration for challenge phase_: For each challenge phase, add the link to the environment image in the [challenge configuration](https://evalai.readthedocs.io/en/latest/configuration.html):
+
+        ```yaml
+        ...
+        challenge_phases:
+            - id: 1
+            ...
+            - environment_image: <docker image uri>
+        ...
+        ```
+
+    Example References:
+    - [Habitat Benchmark](https://github.com/facebookresearch/habitat-lab/blob/b1f2d4791a0065d0791001b72a6c96748a5f9ae0/habitat-lab/habitat/core/benchmark.py): This file contains description of an evaluation class which evaluates agents on the environment.
+    - 
+
+2. **Create a dummy agent**:
+    The participants are expected to submit docker images for their agents which will contain the policy/network and the methods to interact with the environment.
+
+    Like environment, there are a few steps involved in creating the agent:
+    1. _Create a dummy agent script_: In order to help the participants understand what your environment expects, please create a python script for a dummy/baseline/random agent and submit the agent to your challenge/environment for evaluation.
```

Comment:
Remove the word dummy and dummy/baseline/random agent. Say ` Create a starter agent submission`

Reply:
```
and submit the agent to your challenge/environment for evaluation.
```
what does this mean?

Reply:
Fixed. My idea with "submitting the agent to the challenge" was to show how to evaluate locally.